src/lib/geocoding-api/country-codes.ts

```diff
@@ -249,6 +250,257 @@ const countryNameToTwoLetterIsoCountryCodeMap: Record<string, TwoLetterIsoCountr
   Zimbabwe: "ZW",
 };
 
+const countryNameToThreeLetterIsoCountryCodeMap: Record<string, ThreeLetterIsoCountryCode | undefined> = {
```

Comment:
There's a table or something similar in Airtable that we can definitely use in the future. Just bear with me for now, let's take one step in the right direction and then circle back around to this ;)